build/commands/lib/build.js

```diff
@@ -37,6 +37,14 @@ const build = async (buildConfig = config.defaultBuildConfig, options = {}) => {
   config.update(options)
   checkVersionsMatch()
 
+  if (config.useClangCoverage()) {
```

Comment:
I think actually this should move into `buildTargets` because `buildTargets` is sometimes called directly from other commands (like test) and then this step would get skipped. There's a similar issue with `touchOverriddenFiles`, `branding.update()` and `buildNativeRedirectCC`

Reply:
I guess we could follow-up to fix them all, but we should at least open an issue for it

Reply:
and created issue for the remainder: https://github.com/brave/brave-browser/issues/49364